docs/examples/drag-and-drop/README.md

```diff
@@ -0,0 +1,258 @@
+# Drag and Drop
+## Drag and Drop to IPFS and Hypercore
+
+## Introduction
+This tutorial guides you through building a simple yet powerful web application that allows users to drag and drop files, publishing them directly to IPFS and Hypercore. Imagine Alice, an artist who wants to share her digital art easily. With this app, she can drag her artwork into the browser, and it's instantly available on the decentralized web.
+
+## Prerequisites
+- Basic knowledge of HTML, CSS, and JavaScript.
+- Agregore browser installed on your machine.
+
+## Setting Up Your Environment
+1. **Install Agregore Browser:** Download and install the Agregore browser from [Agregore's official website](https://agregore.mauve.moe/).
+2. **Create a Project Folder:** Create a new folder on your computer where you will store your project files.
+
+## Building the Application
+### Step 1: HTML Structure
+Create an `index.html` file in your project folder and add the following HTML structure:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Drag and Drop to IPFS and Hypercore</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main>
+        <label for="protocolSelect">Protocol:</label>
+        <select id="protocolSelect">
+            <option value="ipfs" selected>IPFS</option>
+            <option value="hyper">Hypercore</option>
+        </select>
+        <section id="uploadBox">Drop a file here</section>
+        <ul id="uploadListBox"></ul>
+    </main>
+    <script src="script.js"></script>
+</body>
+</html>
+```
+
+Notice the simplicity of our HTML file.
+- A dropdown element to select the protocol
+- A section to drop the file
+- A list element to contain the URL to our uploaded files
+- Both the CSS and Javascript are loaded from external files for simplicity and maintainability
+
+### Step 2: Adding styles
+Create a styles.css file for styling the application:
+
+```css
+@import url("agregore://theme/vars.css");
+html {
+    background: var(--ag-theme-background);
+    color: var(--ag-theme-text);
+    font-family: var(--ag-theme-font-family);
+}
+#uploadBox {
+    padding: 1em;
+    border: 0.25em solid var(--ag-theme-primary);
+    border-radius: 0.5em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+a {
+    color: var(--ag-theme-secondary);
+    padding:0.15em 0.5em;
+    border-radius: 0.5em;
+}
+a:hover {
+    color: var(--ag-theme-background);
+    background-color: var(--ag-theme-secondary);
+    text-decoration: none;
+}		
+main {
+    margin: 1em;
+}
+select, select * {
```

Comment:
Maybe we can add these to the global agregore theme instead so that we can reuse the style for selects across apps.

Maybe also for the `::selection` and hover options?

https://github.com/AgregoreWeb/agregore-browser/blob/master/app/pages/theme/style.css#L15

Reply:
Created a PR to bring these changes in global agregore theme
https://github.com/AgregoreWeb/agregore-browser/pull/251